wger/exercises/fixtures/muscles.json

```diff
@@ -126,5 +126,29 @@
             "is_front": false, 
             "name": "Triceps brachii"
         }
+    },
+    {
+        "pk": 17,
+        "model": "exercises.muscle",
+        "fields": {
+            "is_front": false,
+            "name": "Lateral Deltoid"
```

Comment:
This muscle, `Lateral Deltoid` is not present in the diagra.

Reply:
Fixed in the recent push. Kindly see the previous comment.